### PR TITLE
Improved documentation

### PR DESCRIPTION
**Pull request by francis-pouatcha**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,43 +1,440 @@
...
+Before proceeding, ensure you have the following tools installed on your system:
+
+* **OpenSSL:** A command-line tool for working with SSL/TLS certificates, keys, and other cryptographic functions.
+* **Keytool:** A Java key and certificate management utility included with the Java Development Kit (JDK).
+
+**Verification:** You can verify that the tools are working by running:
+
+```bash
+openssl version
+keytool -version
```
Comment (ArmandMeppa) [inline Readme.md:22]: Tried this command, but it doesn't work
I propose we change it to
```bash
keytool -h
```

File: Readme.md [hunk 2]
```diff
@@ -1,43 +1,440 @@
...
+* **Java Keytool:** see above
+
+* **OpenSSL:** see above
+
+* **jq (Optional):** `jq` is a handy command-line JSON processor that can simplify some of the configuration tasks in this guide. 
+
+**Verification:** You can verify that the tools are working by running:
 
 ```bash
+keytool -version
```
Comment (ArmandMeppa) [inline Readme.md:54]: Change this too

File: Readme.md (hunk 1 above)
Comment (francis-pouatcha) [reply Readme.md:22]: you mean openssl -h

Comment (francis-pouatcha) [reply Readme.md:22]: both work on my machine

**Review by ArmandMeppa — APPROVED** [review]

**Pull request merged**